backend/de.metas.business/src/main/java/de/metas/inout/impl/InOutDAO.java

```diff
@@ -454,4 +457,23 @@ public <T extends I_M_InOut> Map<InOutId, T> getShipmentsByIds(@NonNull final Se
 				.stream()
 				.collect(ImmutableMap.toImmutableMap(inOut -> InOutId.ofRepoId(inOut.getM_InOut_ID()), Function.identity()));
 	}
+
+	@Override
+	public Collection<InOutAndLineId> retrieveLineIdsForOrderLineIdAvailableForInterimInvoice(@NonNull final OrderLineId orderLine, @Nullable final Object contextProvider)
```

Comment:
why contextProvider param is needed? 
I think we can safely use createQueryBuilder(I_M_InOutLine.class)

Reply:
removed.